node/cmd/ccq/query_server.go

```diff
@@ -209,7 +219,27 @@ func runQueryServer(cmd *cobra.Command, args []string) {
 	signal.Notify(sigterm, syscall.SIGTERM)
 	go func() {
 		<-sigterm
-		logger.Info("Received sigterm. exiting.")
+		if statServer != nil && *shutdownDelay1 != 0 {
```

Comment:
Should this check be >0 because shutdownDelay1 is signed?

Reply:
General comment: Do you need to make sure the delay values are non-negative?

Reply:
Changed it to be a uint.